.tool-versions

```diff
@@ -1,3 +1,3 @@
 erlang 26.2.5
-gleam 1.2.1
```

Comment:
maybe don't update gleam in same PR - I'll do another one to 1.4 which just launched.

Reply:
Sounds good. Thought I'd throw it in there to see what you preferred.